flash/image/classification/input_transform.py

```diff
@@ -50,9 +55,14 @@ def input_per_sample_transform(self):
         return T.Compose([T.ToTensor(), T.Resize(self.image_size), T.Normalize(self.mean, self.std)])
 
     def train_input_per_sample_transform(self):
-        return T.Compose(
-            [T.ToTensor(), T.Resize(self.image_size), T.Normalize(self.mean, self.std), T.RandomHorizontalFlip()]
-        )
+        transforms = [T.TrivialAugmentWide()] if _TORCHVISION_GREATER_EQUAL_0_11 else []
+        transforms += [
+            T.ToTensor(),
+            T.Resize(self.image_size),
+            T.RandomHorizontalFlip(),
+            T.Normalize(self.mean, self.std),
+        ]
```

Comment:
This will move the `TrivialAugmentWide` as the first transform. Is this expected? IMO it will better to apply the augmentation after the normalization.

Reply:
yes, but it does not work as `TrivialAugmentWide` requires uint8 and `ToTensor` convert to float... so this is the only place it can be used...

Reply:
We can get an inspiration from the way it is done in `torchvision`:
https://github.com/pytorch/vision/blob/e288f6ca01a5009ab32f1db010b31f14756e0303/references/classification/presets.py#L18-L41